tests/fields/fields.go

```diff
@@ -17,13 +17,13 @@
 package main
 
 import (
+	"code.google.com/p/goauth2/oauth"
```

Comment:
Shouldn't this one be grouped beyond the blank line with the `"github.com/google/go-github/github"` below anyway?